### PR TITLE
add comments for stat prefix

### PR DESCRIPTION
**Pull request by ramaraochavali**
As discussed in #2405

### CODE REVIEW

File: networking/v1alpha3/virtual_service.proto [hunk 1]
```diff
@@ -1122,7 +1122,10 @@ message HTTPMatchRequest {
 
   // The human readable prefix to use when emitting statistics for this route.
   // The statistics are generated with prefix route.<stat_prefix>.
-  // This should be set for highly critical routes that one wishes to get “per-route” statistics on.
+  // This should be set for highly critical routes that one wishes to get "per-route" statistics on.
```
Comment (ericvn) [inline networking/v1alpha3/virtual_service.proto:1125]: Possibly, the failure in https://github.com/istio/istio.io/pull/11549 is the double quotes being used here. Maybe backticks as in the prior fields?

File: networking/v1beta1/virtual_service.proto [hunk 2]
```diff
@@ -1122,7 +1122,10 @@ message HTTPMatchRequest {
 
   // The human readable prefix to use when emitting statistics for this route.
   // The statistics are generated with prefix route.<stat_prefix>.
-  // This should be set for highly critical routes that one wishes to get “per-route” statistics on.
+  // This should be set for highly critical routes that one wishes to get "per-route" statistics on.
```
Comment (ericvn) [inline networking/v1beta1/virtual_service.proto:1125]: as above

File: networking/v1alpha3/virtual_service.proto (hunk 1 above)
Comment (ramaraochavali) [reply networking/v1alpha3/virtual_service.proto:1125]: The same quotes are used in the other places of file. So it should be fine

File: networking/v1alpha3/virtual_service.proto [hunk 3]
```diff
@@ -1122,7 +1122,10 @@ message HTTPMatchRequest {
...
+  // This prefix is only for proxy-level statistics (envoy_*) and not service-level (istio_*) statistics.
+  // Refer https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#envoy-v3-api-field-config-route-v3-route-stat-prefix
```
Comment (ericvn) [inline networking/v1alpha3/virtual_service.proto:1127]: Refer **to** ...?

Comment (ramaraochavali) [reply networking/v1alpha3/virtual_service.proto:1127]: changed

**Review by ericvn — APPROVED** [review]
LGTM

**Pull request merged**